src/styles/theme.scss

```diff
@@ -23,10 +40,11 @@ $black-pure: #000000;
 $shadow: rgba(89, 97, 116, 0.4);
 
 // Theme colours
-$primary-colour: $blue-darkest;
-$secondary-colour: $blue-light;
-$text-colour: $black;
-$interaction-colour: $purple;
+$primary-colour: $pink;
+$secondary-colour: $grey;
+$text-colour: $black-pure;
+$interaction-colour: $grey;
+$icon-background: $yellow;
 $focus-colour: $lime-green;
```

Comment:
Is that the right focus colour?

Reply:
inherited from CA core. don't know what colour to use here? hopefully, Rose or Holly can offer a better option.

Reply:
Choose one that works with the theme. Pink? or Yellow?